content/blog/2023/12/15/2023-12-15-jenkins-november-newsletter.adoc

```diff
@@ -0,0 +1,111 @@
+Key items include:
+
+* Java 21 support
+* Java 17 or Java 21 recommended
+* Prototype.js JavaScript library removed
+* Drop support for Red Hat Enterprise Linux 7 and its derivatives like CentOS 7
+
+A link:/blog/2023/11/10/Jenkins-Contributor-Summit-in-Brussels/[Jenkins Contributor Summit] will be held in Brussels, Belgium on Friday, February 2, 2024.
+We’ll gather for a day of presentations, planning, and working on the future of Jenkins.
+Immediately after the Contributor Summit, FOSDEM 2024 will be a two day conference focused on open source.
+author:jmmeessen[Jean-Marc Meessen] is gathering link:https://community.jenkins.io/t/jenkins-contributor-summit-on-feb-2-2024-call-for-topics-and-ideas/10689[agenda topics and attendees] on the community site.
+
+
+[[infrastructure]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png[Infrastructure Update]
+
+Contributed by: author:dduportal[Damien Duportal]
+
+Microsoft has donated $40,000 USD to the Continuous Delivery Foundation for use in member projects.
+The Jenkins project is pleased to be using those credits to reduce CDF expenses.
+
+The AWS cost reduction project has a working prototype that is using Cloudflare R2 and the Jenkins mirror system to reduce bandwidth costs.
+We plan to complete the Jenkins Enhancement Proposal, a prototype temporary transition, and the production transition by January 31, 2024.
+
+[[modern-ui]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+Jenkins 2.426.1 provides a new “Appearance” page so that administrators have a single page to configure the Jenkins theme and other user interface details.
+Thanks to author:timja[Tim Jacomb] for the Appearance page.
+
+Jenkins 2.426.1 also updates the Jenkins user experience with confirmation dialogs inside the pages instead of using the web browser’s confirmation dialog.
+Thanks to link:https://github.com/mawinter69[Markus Winter] for the implementation of those dialogs and for the API that allows plugins to do the same.
+
+Keyboard navigation has been improved in Jenkins 2.426.1 as well.
+Users can now access tab panes by keyboard navigation.
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+
+* Java 21 support
+** The https://github.com/jenkinsci/jep/blob/cd176912ed8797ddee1066be59b1a68fb9b6bc77/jep/0000/README.adoc[Jenkins enhancement proposal] by Mark Waite is progressing nicely.
+** Progress of testing the top plugins with Java 21:
+*** 108 of the top plugin repositories passing tests with Java 21.
+*** 20 of the top plugin repositories not yet passing tests with Java 21.
+** The Infrastructure team finished the transition to official 21.0.1+12 Linux (17 on Windows).
```

Comment:
I adjusted the sentence a bit from how it was written previously, but I wasn't sure if there was something else I could have added to make the most sense. @gounthar does this make sense, or would there need to be other adjustments?

Reply:
That's fine with me, thank you Kevin.